quartz/components/renderPage.tsx

```diff
@@ -213,6 +216,17 @@ export function renderPage(
   )
 
   const lang = componentData.fileData.frontmatter?.lang ?? cfg.locale?.split("-")[0] ?? "en"
+
+  var content = <Content {...componentData} />
```

Comment:
Please avoid using `var`. Preferably use `const` or `let` if you have to.

Reply:
Let me fix it